modules/rostests/kmtests/ntos_mm/MmSection.c

```diff
@@ -467,6 +470,22 @@ TestPhysicalMemorySection(VOID)
     RtlCopyMemory(ZeroPageContents, Mapping, PAGE_SIZE);
     MmUnmapIoSpace(Mapping, PAGE_SIZE);
 
+   KeStackAttachProcess(&PsInitialSystemProcess->Pcb, &ApcState);
+#if defined(_M_AMD64) || defined(_M_IX86)
+    CurrentPageDirectory.QuadPart = __readcr3();
+#endif
+    KeUnstackDetachProcess(&ApcState);
+    Mapping = MmMapIoSpace(CurrentPageDirectory, PAGE_SIZE, MmNonCached);
+    ok(Mapping == NULL, "MmMapIoSpace on running system process directory failed, expected NULL got 0x%X\n", (ULONG_PTR)Mapping);
```

Comment:
'on running system process directory' can be moved to a comment.
`s/failed/unexpectedly succeeded/`
`s/%X/%p/`